samples/test/batch-process-document.v1beta3.test.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {Storage} = require('@google-cloud/storage');
+const cp = require('child_process');
+const {assert} = require('chai');
```

Comment:
I would prefer we avoid using `chai` unless we really have a good reason, and stick to `assert`

Reply:
Done, although it's too bad that [Node 13.2 doesn't include assert.match()](https://stackoverflow.com/questions/62496801/nodejs-13-assert-module-missing-methods-like-match).